spec/lib/tasks/data_hygiene/content_item_deduplicator_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+require "tasks/data_hygiene/content_item_deduplicator"
+
+describe Tasks::DataHygiene::ContentItemDeduplicator do
+  let(:fake_stdout) { StringIO.new }
+
+  before do
+    @real_stdout = $stdout
+    $stdout = fake_stdout
+  end
+
+  after { $stdout = @real_stdout }
+
+  describe "#deduplicate" do
+    it "runs without issue" do
+      content_item = create(:content_item_with_content_id, locale: 'cy')
+      dupe = create(:content_item, content_id: content_item.content_id, locale: 'cy')
+      locale_variant = create(:content_item, content_id: content_item.content_id, locale: 'en')
+
+      expect { subject.deduplicate }.to change(ContentItem, :count).by(-1)
+
+      fake_stdout.rewind
+      output = fake_stdout.read
+      expect(output).to match(/2 duplicates found/)
```

Comment:
I like the simplicity of this output matcher but my primary concern (at least for the deduplicate test) is to assert a record has been destroyed, is there a neat way to place multiple matchers around the ```expect { subject.deduplicate }.to ...` ?